### PR TITLE
Prepare the release

### PR DESCRIPTION
**Pull request by samoht**
This version is not yet compatible with latest cohttp and conduit, as I need to make a point release of ocaml-git compatible with it first (I already have a PR, so this should happen soon but I would avoid changing too much things given the big patch in ocaml-git which should land soon).

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+### 1.3.0 (2017-07-27)
+
+**irmin-chunk**
+
+Add a new package: `irmin-chunk`, which was initially in a separate repository
+created by @mounirnasrallah and @samohit and ported to the new Irmin API by
```
Comment (hannesm) [inline CHANGES.md]: typo `@samoht`

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+@g2p (#464)
+
+**irmin-unix**
+
+Re-aadd the `irmin` binary, the example application which used to be
```
Comment (hannesm) [inline CHANGES.md]: typo `Re-add`

**Comment by samoht** [comment]
Thanks @hannesm, I've updated the README

**Comment by samoht** [comment]
The release is blocked on #469 (as the default serialisation format changed with the patches to improve performance of hashing)

**Pull request merged**